contracts/src/interfaces/IRethHyperdriveRead.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IHyperdriveRead } from "./IHyperdriveRead.sol";
+import { IRocketStorage } from "./IRocketStorage.sol";
+import { IRocketTokenRETH } from "./IRocketTokenRETH.sol";
+
+interface IRETHHyperdriveRead is IHyperdriveRead {
```

Comment:
This name looks right, but the filename is still `IRethHyperdriveRead`. The MacOS filesystem is case insensitive, so you might need to use `git mv` to actually update the file name.

Reply:
good catch, I had it updated but git did not register!!